src/ViewRendererBundle/objects/helper/TPkgViewRendererLessCompiler.php

```diff
@@ -191,25 +191,42 @@ public function getGeneratedCssForPortal($portal, $minifyCss = null)
                 $filesForLessParsing[PATH_WEB.$lessFile] = '/';
             }
 
-            \Less_Cache::SetCacheDir($cachedLessDir);
-            try {
-                $cssFile = \Less_Cache::Get($filesForLessParsing, $options);
-            } catch (Exception $exc) {
-                if (false !== strpos($exc->getMessage(), 'stat failed')) {
-                    // Consider this as a 'File removed! Halp!' and clear the cache and try again
-                    array_map('unlink', glob($cachedLessDir.'/*'));
+            $css = '';
 
+            if (_DEVELOPMENT_MODE) {
+                \Less_Cache::SetCacheDir($cachedLessDir);
```

Comment:
If that is not already the case, it would make a lot of sense to me if the `cachedLessDir` is within the `/app/cache` folder somewhere.

Reply:
There is a follow-up issue for this: https://github.com/chameleon-system/chameleon-system/issues/442